conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosIndexer.scala

```diff
@@ -183,6 +193,30 @@ class TezosIndexer private (
       } else emptyOutcome
   }
 
+  private def processLastForks(
+      maxIndexedLevel: BlockLevel,
+      depth: Long,
+      interval: Long,
+      iteration: Long
+  ): Future[Option[AccountResetEvents]] = {
+    lazy val emptyOutcome = Future.successful(Option.empty)
+    if (featureFlags.forkHandlingIsOn && maxIndexedLevel != indexedData.defaultBlockLevel && iteration % interval == 0)
+      backtrackingForkProcessor.handleForkFrom(maxIndexedLevel, depth).flatMap {
+        case None =>
+          println(s"AFH: No fork detected up to $maxIndexedLevel")
```

Comment:
"fork" => "local fork" for clarity.

Reply:
changed, also changed to use logger(I turned off logging for testing and used printlns to see better)